src/components/Map/Map.tsx

```diff
@@ -185,6 +185,16 @@ function Map({
           placeholder="Enter zip code..."
           value={zipcode}
           onChange={e => setZipcode(e.target.value)}
+          onKeyDown={(e) => {
```

Comment:
Adrain, if we could chat about this I've never gotten a onKeyDown to work when I've tried it.